x/gamm/keeper/pool.go

```diff
@@ -22,8 +22,8 @@ func (k Keeper) UnmarshalPool(bz []byte) (types.PoolI, error) {
 	return acc, k.cdc.UnmarshalInterface(bz, &acc)
 }
 
-// GetPoolAndPoke returns a PoolI based on it's identifier if one exists. Prior
-// to returning the pool, the weights of the pool are updated via PokePool.
+// GetPool returns a PoolI based on it's identifier if one exists. If poolId corresponds
+// to a pool with weights (e.g. balancer), the weights of the pool are updated via PokePool prior to returning.
```

Comment:
What do you think about just keeping the name? It seems like its easier in terms of API to understand that the poke is a no-op for stableswap rather than leaving poke out of the name and having to look into the function to see it also pokes. No strong opinion either way so close as desired.

Reply:
whoops forgot to change this back after trying things but yeah should be `GetPoolAndPoke`